content/en/docs/multi-center/modules/hosp/admissions.md

```diff
@@ -0,0 +1,145 @@
+| Admitted to L&D                                        | Admitted to Labor and Delivery                |
+| Planned Readmission - DC/transferred to nursing home (custodial) |                     |
+| unknown                                                |                                 |
+| Cancer Center or Children's Hospital                   |                                 |
+| Home with Outpatient Services                          |                                 |
+| Critical Access Hospital                               |                                 |
+| Planned Readmission - DC/transferred to other type of healthcare institution |     |
+| Gift of Hope / Still a Patient                         |                                 |
+| Nursing Home (Custodial)                               |                                 |
+| Home with Hospice                                      |                                 |
+| VA System Facility                                     |                                 |
+| Planned Readmission - DC/transferred to Long-term Acute Care Hospital (LTAC) |       |
+| Swing Bed                                              |                                 |
+| Against Medical Advice (AMA) or Elopement              |                                 |
+| Skilled Nursing Facility or Subacute Rehab Care        |                                 |
+| Designated Disaster Alternative Care Site               |                                 |
+| Acute Care Hospital                                    |                                 |
+| Long-Term Acute Care Hospital (LTAC)                   |                                 |
+
+
+### `insurance`, `language`, `marital_status`, `race`, `ethnicity`
+
+The `insurance`, `language`, `marital_status`, and `race` and `ethnicity` columns provide information about patient demographics for the given hospitalization. Note, in BIDMC there is only one column for `race`, however we have added `ethnicity` column to incorporate NMHC's data.
+
+The race column in NMHC includes:
+
+- American Indian or Alaska Native
+- Other
+- Unknown
+- 2 or more races
+- Unable to Answer
+- Native Hawaiian or Other Pacific Islander
+- Asian
+- White
+- Declined
+- Black or African American
+
+The ethnicity column in NMHC includes:
+
+- Not Hispanic or Latino
+- Hispanic or Latino
+- Declined
+- Unable to Answer
+
+### `edouttime`
+
+The date and time at which arrival of the patient in the emergency department was registered.
+
+### `edouttime`
+The date and time at which the patient was discharged from the emergency department, either discharged from the hospital or transferred.
+
+### `hospital_expire_flag`
+
+This is a binary flag which indicates whether the patient died within the given hospitalization. `1` indicates death in the hospital as noted in the `dod` column as part of the *patient* table, and `0` indicates survival to hospital discharge. 
```

Comment:
was this death in the hospital or death at any time? this column should be death within the associated hospital admission.

Reply:
thanks for the suggestion, added a more explicit statement to address death within hospital as presented by the binary flag.